step-capstone/src/components/Map.js

```diff
@@ -0,0 +1,42 @@
+import React, { createRef } from 'react'
```

Comment:
Something has gone wrong with the diff here, I'm seeing this entire file as new which I know is not the case. You might need to pull from master.

Reply:
fixed